Mage/src/main/java/mage/game/permanent/token/SquidToken.java

```diff
@@ -5,30 +5,32 @@
 import mage.constants.SubType;
 import mage.MageInt;
 import mage.abilities.keyword.IslandwalkAbility;
+import java.util.Arrays;
 
 /**
  *
  * @author spjspj
  */
-public final class ChasmSkulkerSquidToken extends TokenImpl {
+public final class SquidToken extends TokenImpl {
 
-    public ChasmSkulkerSquidToken() {
-        super("Squid Token", "1/1 blue Squid creature token with islandwalk");
-        this.setOriginalExpansionSetCode("M15");
+    public SquidToken() {
+        super("Squid Token", "1/1 blue Squid creature token with Islandwalk");
         cardType.add(CardType.CREATURE);
         color.setBlue(true);
         subtype.add(SubType.SQUID);
         power = new MageInt(1);
         toughness = new MageInt(1);
 
         this.addAbility(new IslandwalkAbility());
+
+        availableImageSetCodes = Arrays.asList("M15", "C16");
     }
 
-    public ChasmSkulkerSquidToken(final ChasmSkulkerSquidToken token) {
+    public SquidToken(final SquidToken token) {
```

Comment:
Does this have to be public?

Reply:
It being public is a remainder from the previous rendition of the code. I think you're right that it could/should be private.

Reply:
Everything else looks good. Do you want push a commit changing it to private, or leave that for a separate commit where you grab the other tokens too?

Reply:
I'll quickly push a commit making it private before it gets merged

Reply:
Should be good to go now.